spec/index.bs

```diff
@@ -2008,9 +2008,15 @@ conditions.
 <script type=idl>
 dictionary GPUShaderModuleDescriptor : GPUObjectDescriptorBase {
     required DOMString code;
+    object? sourceMap = null;
 };
 </script>
 
+{{GPUShaderModuleDescriptor/sourceMap}}, if non-null, is interpreted as a source-map-v3 format.
```

Comment:
Dictionary members are optional by default:

```suggestion
    object sourceMap;
};
</script>

{{GPUShaderModuleDescriptor/sourceMap}}, if defined, is interpreted as a source-map-v3 format.
```

(here I'm using "defined" to mean "both set and not `undefined`". I don't know what the standard wording is.)

Reply:
"defined" sounds accurate